actions/qemuimg2disk/v1/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM alpine:3.13.5 as builder
+ENV QEMU_NAME="qemu"
+ENV QEMU_REV="6.0.0-rc3"
+ENV QEMU_SRC_BASENAME="${QEMU_NAME}-${QEMU_REV}"
+ENV QEMU_SRC_URL="https://download.qemu.org/${QEMU_SRC_BASENAME}.tar.xz"
+
+RUN apk update
+RUN apk upgrade
```

Comment:
This is ineffective, the alpine container images don't have the directories on disk where these files would be stored so its only updating the process's in memory info.